components/card/__tests__/__snapshots__/demo.test.js.snap

```diff
@@ -556,7 +556,7 @@ exports[`renders ./components/card/demo/loading.md correctly 1`] = `
           class="ant-skeleton-header"
         >
           <span
-            class="ant-skeleton-avatar ant-skeleton-avatar-lg ant-skeleton-avatar-circle"
+            class="ant-skeleton-avatar ant-skeleton-avatar-circle"
```

Comment:
Is that possible keep the class?

Reply:
I guess we can keep this class.  But I think use large  as default size not make sense.

Could you please explain why we need large size?

Reply:
It's by design and currently we may not change this: https://github.com/ant-design/ant-design/issues/10308#issuecomment-403429215

Reply:
> It's by design and currently we may not change this: [#10308 (comment)](https://github.com/ant-design/ant-design/issues/10308#issuecomment-403429215)

fixed.